libraries/AP_BoardConfig/AP_BoardConfig.cpp

```diff
@@ -378,6 +378,16 @@ const AP_Param::GroupInfo AP_BoardConfig::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("IO_DSHOT", 28, AP_BoardConfig, state.io_dshot, 0),
 #endif
+
+#if AP_CRASHDUMP_ACK_ENABLED
+    // @Param: CRSHDMP_ACK
+    // @DisplayName: CrashDump Acknowledgement Required
```

Comment:
I'd remove the "Required" at the end.

Reply:
FS_CRASH_CHECK already exists as a parameter, suggest using CRASH not CRSH so that a search for CRASH will return both parameters.

Reply:
Reworded this to "Disable CrashDump Arming Check"